### PR TITLE
Feature/adjuster ui

### PR DESCRIPTION
**Pull request by Tallmaris**

### CODE REVIEW

**Comment by Tallmaris** [comment]
Hi @tznind , happy to have a quick chat about the advanced UI. Removing the option makes the simple UI nicer and avoid a cross over of functionalities (being both an UI for importing a file and for editing pipelines and gives better reporting). Clearly we may need to find a way to reintroduce the advanced UI somehow

**Comment by tznind** [comment]
Cool as long as advanced isn't gone for good. it does make sense.

File: Rdmp.UI/Rdmp.UI.csproj [hunk 1]
```diff
@@ -11,6 +11,10 @@
     <LangVersion>7.1</LangVersion>
     <GenerateResourceUsePreserializedResources>true</GenerateResourceUsePreserializedResources>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```
Comment (tznind) [inline Rdmp.UI/Rdmp.UI.csproj:14]: Can we get this on all csproj? maybe outside the PropertyGroup tag too?

File: Rdmp.UI/SimpleDialogs/SimpleFileImporting/CreateNewCatalogueByImportingFileUI.cs [hunk 2]
```diff
@@ -103,7 +103,7 @@ private void BuildHelpFlow()
         private void btnBrowse_Click(object sender, EventArgs e)
         {
             OpenFileDialog ofd = new OpenFileDialog();
-            ofd.Filter = "Comma Separated Values|*.csv|Excel File|*.xls*|All Files (Advanced UI Only)|*.*";
+            ofd.Filter = "Comma Separated Values|*.csv|Excel File|*.xls*|Text File|*.txt|All Files|*.*";
```
Comment (tznind) [inline Rdmp.UI/SimpleDialogs/SimpleFileImporting/CreateNewCatalogueByImportingFileUI.cs:106]: Does this .txt lpay well with pipeline auto selection in this UI?

File: Rdmp.UI/Rdmp.UI.csproj (hunk 1 above)
Comment (Tallmaris) [reply Rdmp.UI/Rdmp.UI.csproj:14]: I thought it was set to "Portable" on purpose to cope with Linux builds... but ok I can put it in for all csproj.

File: Rdmp.UI/SimpleDialogs/SimpleFileImporting/CreateNewCatalogueByImportingFileUI.cs (hunk 2 above)
Comment (Tallmaris) [reply Rdmp.UI/SimpleDialogs/SimpleFileImporting/CreateNewCatalogueByImportingFileUI.cs:106]: It does now... totally forgot about that

**Pull request merged**